user-registration-app/src/app/views/live-workshop/live-workshop.component.ts

```diff
@@ -41,6 +41,11 @@ export class LiveWorkshopComponent implements OnInit {
     return this._endTime.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
   }
 
+  // Maybe not the best way to detect if it's rich text or not?
+  isRichTextDescription(): boolean {
+    return this.description.charAt(0) === "<";
```

Comment:
Bruh, could probably copy some HTML regex from StackOverflow

Reply:
that'd be too smart tho